Framework/Crystal/src/SaveIsawPeaks.cpp

```diff
@@ -373,6 +395,41 @@ void SaveIsawPeaks::exec() {
   out.close();
 }
 
+bool SaveIsawPeaks::bankMasked(IComponent_const_sptr parent,
+                               const DetectorInfo &detectorInfo) {
```

Comment:
Just a quick comment: Have a look at https://github.com/mantidproject/mantid/blob/master/Framework/API/inc/MantidAPI/ComponentInfo.h#L62. This function might simplify to something like

```cpp
for (auto detIndex : ws.componentInfo().detectorIndices(compIndex)) {
  if detInfo.isMasked(detIndex);
}
```